src/configuration.ts

```diff
@@ -207,35 +208,35 @@ export class ConnectionConfiguration implements IConnectionConfiguration {
 
   get pdkRubyLib(): string {
     var lib = path.join(this.puppetBaseDir, 'lib');
-    if (process.platform === 'win32') {
-      // Translate all slashes to / style to avoid puppet/ruby issue #11930
-      lib = lib.replace(/\\/g, '/');
-    }
+    lib = this.replaceSlashes(lib);
     return lib;
   }
 
   get pdkRubyVerDir(): string {
     var rootDir = path.join(this.puppetBaseDir, 'private', 'puppet', 'ruby');
 
-    return path.join(rootDir, this.gemRubyVersionDir);
```

Comment:
Don't forget to delete the references to `gemRubyVersionDir` and `rubyVersionDir` in the class definition and constructor.

Reply:
Thanks for the reminder